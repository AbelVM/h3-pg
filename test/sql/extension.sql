\pset tuples_only on

--
-- TEST h3_get_extension_version
--

SELECT h3_get_extension_version();