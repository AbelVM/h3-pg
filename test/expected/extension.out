\pset tuples_only on
--
-- TEST h3_get_extension_version
--
SELECT h3_get_extension_version();
 1.0.0

